src/borg/helpers.py

```diff
@@ -2277,3 +2277,10 @@ def dash_open(path, mode):
 
 def is_terminal(fd=sys.stdout):
     return hasattr(fd, 'isatty') and fd.isatty() and (sys.platform != 'win32' or 'ANSICON' in os.environ)
+
+
+def umount(mountpoint):
+    try:
+        return subprocess.call(['fusermount', '-u', mountpoint])
```

Comment:
For a single-use four-line function I consider it "prettier" to just define it in one place, rather than spreading it out across 6+ modules.